proto-lens-tests/tests/proto3_test.hs

```diff
@@ -98,6 +98,26 @@ main = testMain
       , testCase "string" $ (def :: Strings) @=? (def & string .~ "")
       , testCase "enum" $ (def :: Foo) @=? (def & enum .~ Foo'Enum1)
       ]
+  -- Enums are all pattern aliases
+  , testGroup "enum"
+      [ testCase "aliases are exported" $ Foo'Enum2 @=? Foo'Enum2a
+      , testCase "enum values" $ do
+          map toEnum [0, 3, 3] @=? [Foo'Enum1, Foo'Enum2, Foo'Enum2a]
+          ["Foo'Enum1", "Foo'Enum2", "Foo'Enum2", "toEnum 5"]
+              @=? map show [Foo'Enum1, Foo'Enum2, Foo'Enum2a, toEnum 5]
```

Comment:
is `toEnum` the only way to construct lax enums?

Reply:
We're also exporting the newtype constructor itself.